### PR TITLE
Draft of simple Python library

### PR DESCRIPTION
**Pull request by mrocklin**
cc @quasiben

### CODE REVIEW

File: Dockerfile-master [hunk 1]
```diff
@@ -2,12 +2,12 @@ FROM ubuntu:14.04
 
 ENV LANG C.UTF-8
 
-RUN apt-get update -qq
-RUN apt-get install curl bzip2 -y -qq
+RUN apt-get update
+RUN apt-get install curl bzip2 -y
```
Comment (mrocklin) [inline Dockerfile-master]: I removed the -qq flag here to make this match with the `Dockerfile-slave` file in hopes that we can avoid some setup time through deduplication.

File: Dockerfile-master [hunk 2]
```diff
@@ -21,5 +21,3 @@ EXPOSE 6446
 ENV DRMAA_LIBRARY_PATH /usr/lib/gridengine-drmaa/lib/libdrmaa.so
 ENV SGE_ROOT /var/lib/gridengine/
 ENV SGE_CELL default
-
-CMD python -m SimpleHTTPServer
```
Comment (mrocklin) [inline Dockerfile-master]: Was this necessary from before?  Or was it here for testing

File: docker-compose.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@ services:
       context: .
       dockerfile: Dockerfile-slave
     container_name: slave_one
-    hostname: slave_one.daskdrmaa_default
+    hostname: slave_one
```
Comment (mrocklin) [inline docker-compose.yml]: Are these namechanges ok?

File: Dockerfile-master (hunk 2 above)
Comment (quasiben) [inline Dockerfile-master]: It was there for testing

File: docker-compose.yml (hunk 3 above)
Comment (quasiben) [inline docker-compose.yml]: I recall the odd hostname being critical, I'll test shortly

Comment (quasiben) [inline docker-compose.yml]: Nevermind, things are working locally for me and looks like test are passing

**Comment by quasiben** [comment]
This starts 2 workers and the scheduler on the same host, correct ?

**Comment by mrocklin** [comment]
It starts two workers, it doesn't guarantee which host.

**Pull request merged**